src/Ninja/G1CP/Content/Fixes/Session/fix188_CorKalomWeedExploit.d

```diff
@@ -0,0 +1,44 @@
+/*
+ * #188 Cor Kalom can be told that the weed was delivered multiple times
+ */
+func int G1CP_188_CorKalomWeedExploit() {
+    var int symbId;
+    if (G1CP_IsFixApplied(24)) {
+        symbId = MEM_GetSymbolIndex("Kalom_Krautbote");
+    } else {
+        symbId = MEM_GetSymbolIndex("Kalom_DrugMonopol");
+    };
+    if (MEM_GetSymbolIndex("Info_Kalom_KrautboteBACK_Condition") != -1)
+    && (symbId != -1) {
+        HookDaedalusFuncS("Info_Kalom_KrautboteBACK_Condition", "G1CP_188_CorKalomWeedExploit_Hook");
+        return TRUE;
+    } else {
+        return FALSE;
+    };
+};
+
+/*
+ * This function intercepts the dialog condition to introduce more conditions
+ */
+func int G1CP_188_CorKalomWeedExploit_Hook() {
+    G1CP_ReportFuncToSpy();
+
+    // Define possibly missing symbols locally
+    const int LOG_SUCCESS = 2;
+
+    // Check if quest is running
+    var int symbPtr; 
+    if (G1CP_IsFixApplied(24)) {
+        symbPtr = MEM_GetSymbol("Kalom_Krautbote");
+    } else {
+        symbPtr = MEM_GetSymbol("Kalom_DrugMonopol");
+    };
+    if (symbPtr) {
+        if (MEM_ReadInt(symbPtr + zCParSymbol_content_offset) == LOG_SUCCESS) {
```

Comment:
I suggest taking the inverse of
```D
if (var == LOG_RUNNING) {
    return TRUE;
};
```
So instead I propose to turn this condition here into

```D
if (var != LOG_RUNNING) {
    return TRUE;
};
```